chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/ApiKeyDTO.java

```diff
@@ -4,131 +4,36 @@
 import java.util.Date;
 
 import gov.healthit.chpl.entity.ApiKeyEntity;
-import gov.healthit.chpl.util.Util;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
+@NoArgsConstructor
+@Data
 public class ApiKeyDTO implements Serializable {
     private static final long serialVersionUID = 7091753452944248313L;
     private Long id;
     private String apiKey;
     private String email;
     private String nameOrganization;
-    private Boolean whitelisted;
+    private Boolean unrestricted;
     private Date creationDate;
     private Date lastModifiedDate;
     private Long lastModifiedUser;
     private Boolean deleted;
     private Date lastUsedDate;
     private Date deleteWarningSentDate;
 
-    /**
-     * Default constructor.
-     */
-    public ApiKeyDTO() {
-    }
-
-    /**
-     * Constructed from an entity.
-     * @param entity the entity
-     */
-    public ApiKeyDTO(final ApiKeyEntity entity) {
+    public ApiKeyDTO(ApiKeyEntity entity) {
         this.id = entity.getId();
         this.apiKey = entity.getApiKey();
         this.email = entity.getEmail();
         this.nameOrganization = entity.getNameOrganization();
-        this.whitelisted = entity.getWhitelisted();
+        this.unrestricted = entity.getUnrestricted();
         this.creationDate = entity.getCreationDate();
         this.lastModifiedDate = entity.getLastModifiedDate();
         this.lastModifiedUser = entity.getLastModifiedUser();
         this.deleted = entity.getDeleted();
         this.setLastUsedDate(entity.getLastUsedDate());
         this.setDeleteWarningSentDate(entity.getDeleteWarningSentDate());
     }
-
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(final Long id) {
-        this.id = id;
-    }
-
-    public String getApiKey() {
-        return apiKey;
-    }
-
-    public void setApiKey(final String apiKey) {
-        this.apiKey = apiKey;
-    }
-
-    public String getEmail() {
-        return email;
-    }
-
-    public void setEmail(final String email) {
-        this.email = email;
-    }
-
-    public String getNameOrganization() {
-        return nameOrganization;
-    }
-
-    public void setNameOrganization(final String nameOrganization) {
-        this.nameOrganization = nameOrganization;
-    }
-
-    public Boolean getWhitelisted() {
-        return whitelisted;
-    }
-
-    public void setWhitelisted(final Boolean whitelisted) {
-        this.whitelisted = whitelisted;
-    }
-
-    public Date getCreationDate() {
-        return Util.getNewDate(creationDate);
```

Comment:
I think this line might need to be restored. Not sure exactly what the function does, or if it's actually needed, but using the Lombok thing would change functionality. Same thing with the other dates